netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java

```diff
@@ -1302,6 +1320,7 @@ public void maxRstCount_exceedsLimit_fails() throws Exception {
     manualSetUp();
     assertThrows(ClosedChannelException.class, () -> rapidReset(maxRstCount + 1));
     assertFalse(channel().isOpen());
+    channel().releaseOutbound();
```

Comment:
I don't see any outbound messages in the channel when this is called. Check here and in other places whether it is really necessary. 
Also since you also have it in _tearDown_ does it not suffice already?

Reply:
@kannanjgithub I can see even with tearDown() method the channel().releaseOutbound() entry is required as I have validated the multiple UT's without this outbound messages are not getting released. please find the below snippet for one of UT which says messages are not released without this entry and its printing the count of the same.

![image](https://github.com/user-attachments/assets/ce1f8d20-a7c0-466a-995c-de066b3c156e)

Reply:
@kannanjgithub As per our recent discussion, Removed the channel().releaseOutbound() entries in the UT's which are calling the rapidReset() method as its already handled here.

Also Reverted back the same in other UTs which is not required.